File: CursoCsharpUdemy/exer/outros/Exer16.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer16
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Informe o primeiro número: ");
            int number1 = int.Parse(Console.ReadLine());

            Console.WriteLine("Informe o segundo número: ");
            int number2 = int.Parse(Console.ReadLine());

            if (number1 > number2)
            {
                Console.WriteLine("Primeiro número maior que segundo.");
            }
            else if (number1 < number2)
            {
                Console.WriteLine("Segundo número maior que primeiro.");
            }
            else
            {
                Console.WriteLine("São iguais.");
            }
        }
    }
}

File: CursoJava/old/Curso de Java/Exer/src/lista64Exer/Exer16.java
import java.util.Scanner;

public class Exer16 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Informe o primeiro número: ");
        int number1 = sc.nextInt();

        System.out.print("Informe o segundo número: ");
        int number2 = sc.nextInt();

        if (number1 > number2) {
            System.out.println(number1 + " é maior que " + number2);
        } else if (number1 < number2) {
            System.out.println(number2 + " é maior que " + number1);
        } else {
            System.out.println("Eles são iguais");
        }

        sc.close();
    }

}
